Automattic/wp-calypso
# Fixes the style for the Blaze dashboard

Related to https://jira.tumblr.net/browse/SELFSERVE-403

We have a problem now in production where the mobile version of the Blaze dashboard page is showing the post meta info incorrectly:

![Screenshot 2023-04-26 at 18 57 51](https://user-images.githubusercontent.com/1258162/234712297-8b31baa2-4f6f-4767-a2b0-a14f22a60600.png)

## Proposed Changes

This PR isolates the style for the redesigned Blaze dashboard page from the current Blaze dashboard. We had conflicts with the class names between both modules because we used the same class name prefix.

## Testing Instructions

**Testing the redesign (development only)**

1. Run Calypso locally,
2. Login as A8C user,
3. Check out Tools > Advertising 
4. Verify that the page is showing the new design

<img width="1100" alt="233160240-e3feb673-1595-4b17-90f3-b45cc3bf4697" src="https://user-images.githubusercontent.com/1258162/234711738-e1ce0b89-ff32-4c6a-87e3-7e14db532085.png">

**Testing the current page**

1. Open the Live preview
2. Login as A8C user,
3. Check out Tools > Advertising 
4. Change the viewport to a mobile device
5. Verify that the mobile view of the page is displayed correctly

![Screenshot 2023-04-26 at 18 56 39](https://user-images.githubusercontent.com/1258162/234712044-eda97ad1-b5ec-4039-939f-e30f12fdffa1.png)





## Pre-merge Checklist
